dali/operators/generic/join.cc

```diff
@@ -42,11 +44,13 @@ DALI_SCHEMA(Stack)
   .DocStr(R"(Joins the input tensors along a new axis.
 
 The shapes of respective tensors in the inputs must match.)")
-  .AddOptionalArg<int>("axis", R"(The axis in the output tensor along which the inputs are stacked.
+  .AddOptionalArg<int>("axis", R"code(The axis in the output tensor along which the inputs are stacked.
 
 The axis is inserted before a corresponding axis in the inputs. A value of 0 indicates that whole
 tensors are stacked. Specifying ``axis`` equal to the number of dimensions in the inputs causes
-the values from the inputs to be interleaved)", 0, false)
+the values from the inputs to be interleaved).
+
+Accepted range is [-ndim, ndim], with negative indices are counted from the back.)code", 0, false)
```

Comment:
Same as above, I think the `are` does not fit here. Or does it? 

```suggestion
Accepted range is [-ndim, ndim]. Negative indices are counted from the back.)code", 0, false)
```

or 

```suggestion
Accepted range is [-ndim, ndim], with negative indices counted from the back.)code", 0, false)
```